examples/notebooks/Accessing C Struct Data.ipynb

```diff
@@ -0,0 +1,327 @@
+{
```

Comment:
Could this file name used `_` instead of spaces?

Reply:
There are other files in that directory that are using spaces.  The notebook filename is also the title of the notebook.  Using spaces are common.